04week/loop.js

```diff
@@ -31,6 +31,12 @@ for(const val in persons){
   console.log(persons[val]);
 }
 
+for(const val in persons){
+	if(persons[val] === 'birthDate'){
```

Comment:
Almost, but not quite! A for...in loop iterates through the keys (left side) of the key value pairs in an object, not the values (right side).